pkg/controller/datavolume/clone-controller-base.go

```diff
@@ -524,7 +524,7 @@ func isCrossNamespaceClone(dv *cdiv1.DataVolume) bool {
 }
 
 // addCloneWithoutSourceWatch reconciles clones created without source once the matching PVC is created
-func addCloneWithoutSourceWatch(mgr manager.Manager, datavolumeController controller.Controller, typeToWatch client.Object, indexingKey string) error {
+func addCloneWithoutSourceWatch(mgr manager.Manager, datavolumeController controller.Controller, typeToWatch client.Object, indexingKey string, op dataVolumeOp) error {
```

Comment:
What does `Op` mean?

Reply:
It's the datavolume requested operation so one of these
https://github.com/kubevirt/containerized-data-importer/blob/c51229623bd2fb68acca6060291bfebcd6400ede/pkg/controller/datavolume/controller-base.go#L157-L164

Reply:
So `op` is consistent across the project, should I refactor?